### PR TITLE
standarizing logger on iOS

### PR DESCRIPTION
**Pull request by aradiquez**
//cc @Lucashuang0802 @marinaserranomontes @marko07 @adrice727 @albertoacn

### CODE REVIEW

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m [hunk 1]
```diff
@@ -19,9 +19,11 @@
 NSString* const KLogClientVersion = @"ios-vsol-1.0.0";
 NSString* const KLogActionInitialize = @"Init";
 NSString* const KLogActionStart = @"Start";
-NSString* const KLogActionStop = @"Stop";
-NSString* const KLogActionSendMessage = @"SendMessage";
-NSString* const KLogActionReceiveMessage = @"ReceiveMessage";
+NSString* const KLogActionStop = @"End";
+NSString* const KLogActionOpen = @"OpenTC";
+NSString* const KLogActionClose = @"CloseTC";
+NSString* const KLogActionSendMessage = @"Send Msg";
```
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m]: This content is different with the one from Android.

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m [hunk 2]
```diff
@@ -19,9 +19,11 @@
...
+NSString* const KLogActionReceiveMessage = @"Receive Msg";
```
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m]: This content is different with the one from Android.

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m [hunk 3]
```diff
@@ -211,19 +211,22 @@ - (void)disconnect {
 }
 
 - (void)show {
-    
+    [OTKLogger logEventAction:KLogActionOpen variation:KLogVariationAttempt completion:nil];
```
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m]: If you have the attempt action, we should have success and failure.

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m [hunk 4]
```diff
@@ -211,19 +211,22 @@ - (void)disconnect {
...
     
     UIViewController *topViewController = [UIViewController topViewControllerWithRootViewController];
     if (topViewController) {
         [topViewController.view addSubview:self];
+        [OTKLogger logEventAction:KLogActionOpen variation:KLogVariationSuccess completion:nil];
     }
 }
 
 - (void)dismiss {
+    [OTKLogger logEventAction:KLogActionClose variation:KLogVariationAttempt completion:nil];
```
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m]: If you have the attempt action, we should have success and failure.

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m (hunk 2 above)
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m]: !completed

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m (hunk 1 above)
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m]: !completed

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m (hunk 3 above)
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m]: Let's put success only, there is no failure at all.

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m (hunk 4 above)
Comment (aradiquez) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m]: !resolve

Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m]: Let's put success only, there is no failure at all.

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m [hunk 5]
```diff
@@ -19,9 +19,11 @@
 NSString* const KLogClientVersion = @"ios-vsol-1.0.0";
 NSString* const KLogActionInitialize = @"Init";
 NSString* const KLogActionStart = @"Start";
-NSString* const KLogActionStop = @"Stop";
-NSString* const KLogActionSendMessage = @"SendMessage";
-NSString* const KLogActionReceiveMessage = @"ReceiveMessage";
+NSString* const KLogActionStop = @"End";
```
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m]: Can you change the const name as well to match with the content?

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m (hunk 4 above)
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m]: !completed

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m (hunk 3 above)
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextChatView/OTTextChatView.m]: !completed

File: ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m (hunk 5 above)
Comment (Lucashuang0802) [inline ios/TextChatAccPackKit/OTTextChatKit/OTTextMessage/OTTextMessageManager.m]: !completed

**Pull request merged**